common/gce.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module for using the Google Compute Engine (GCE) API."""
+import threading
+
+import dateutil.parser
+
+from googleapiclient import discovery
+from oauth2client.client import GoogleCredentials
+
+thread_local = threading.local()  # pylint: disable=invalid-name
```

Comment:
Why not just
_THREAD_LOCAL
why we disable invalid name instead of naming them to avoid lint issue.

Reply:
Because the linter is wrong. This variable is not a constant and therefore should not be capitalized.